package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "math_magicians",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
```

Comment:
- [ ] Great job so far. You package.json is not having the correct-react scripts and as such I cannot start your application. Get the correct react scripts. This scripts are normally generated automatically if you use the [right command](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) to create your new react app.
Your scripts should be similar to those below:
```
"scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
```